### DISCUSSION TITLE
Welcome to DataSaturdays Discussions!

### DISCUSSION THREAD

**Discussion by SQLDBAWithABeard** [dataplat/DataSaturdays#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
  
## Culture
We expect and demand that you follow some basic rules. Nothing dramatic here. There will be a proper code of conduct for the websites added soon, but in this repository

**BE EXCELLENT TO EACH OTHER**

Do I need to say more? If your behaviour or communication does not fit into this statement, we do not wish for you to help us.

**Comment by felipeschneider88** [comment]
Hello, my name y Felipe Schneider. I'm a SQL Server DBA. I love all related to databases. Now I'm learning about cockroachDB and DevOps in databases :) I have experience with .NET, python, JAVA and Powershell
